sycl/test/basic_tests/generic_type_traits.cpp

```diff
@@ -99,41 +99,48 @@ int main() {
   is_genint
   */
 
+  // According to SYCL 2020 specification, revision 6 section 4.14.2.2. Aliases:
```

Comment:
Note for reviewers: those generic `is_ulong`, `is_longn`, `is_genlong`, `is_ugenlong` type traits are not used anywhere except for this test. And there seem to be no built-ins which are defined through `longn`, `ulongn` in the spec either.

So, we can consider removing those completely instead of fixing this test

Reply:
I think it makes sense to remove them then.

Reply:
And I've just removed it in 441a8745c4c42bb65dd26ca7538ef9eacc4f1534. @sergey-semenov, could you please take another look?

Reply:
Looks good, thanks!